Misc/NEWS.d/next/Library/2019-12-07-18-58-44.bpo-38994.IJYhz_.rst

```diff
@@ -1 +1 @@
-Implement ``__class_getitem__`` for ``os.PathLike``, ``pathlib.Path``
+Implement ``__class_getitem__`` for ``os.PathLike``, ``pathlib.PurePath``. Patch by Batuhan Taskaya.
```

Comment:
This change looks unrelated.

Reply:
It was a mistake from a previous patch and I didn't want to open a new PR for that. I explained it in PR body.

Reply:
OK, anyway, it looks like there is a merge conflict now, you need to fix it before we can move on.

Reply:
Thanks for the reviews. I'm applying them.